documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,90 @@
+# WCA Competition Requirements Policy
+<div class="version">
+### Version 1.0
+</div>
+
+## Purpose
+The purpose of this policy is to define the requirements and processes that a competition must meet and follow in order to be recognized as a WCA Competition.
+
+## Policy
+### WCA Competitions
+An official WCA Competition must:
+
+1. Be announced at least 28 days (four weeks) before the start of the competition on the WCA website.
+   1. WCA Delegates should allow a suitable time frame for competitions to be announced after they are confirmed to the WCA Competition Announcement Team.
+2. Comply with the [WCA Regulations](wca{regulations}).
+3. Comply with all [WCA Competition Policies](wca{documents}).
+4. Be publicly accessible.
+5. Be open to all competitors and spectators who wish to attend, up to the maximum capacity stated on the website or otherwise permitted by the venue.
+6. Be held on a single date or on two or more consecutive days.
+7. Have at least one official event.
+8. Have at least one competitor.
+   1. The competition should have at least 12 competitors.
+9. Have at least one designated WCA Delegate in attendance.
+   1. The WCA Delegate for a competition is responsible for:
+      1. Submitting the Delegate Report to the WCA Regulations Committee and the WCA Quality Assurance Committee regarding adherence to WCA Regulations and quality standards during the competition, the overall course of the competition, and any incidents that happened. The report must be submitted within one week of the end of the competition.
+      2. Sending the competition results to the WCA Results Team within one week of the end of the competition.
+         1. Results should be ready at the end of the last day of competition. Corrections to the results should be available within one week of the competition date.
```

Comment:
This really doesn't reflect reality these days. Delegates are actually expected to double-check results before sending them in. Another soon to be merged commit will also introduce a major shift as several result checks will need to be passed before a result submission is even considered successful.

My Proposal: 
- "Submitting the competition results ..." instead of "Sending the competition results ..."
- Remove "Results should be ready at the end of the last day of competition. Corrections to the results should be available within one week of the competition date." 
- Add "Results should be double-checked and free of errors before submission." 
- Add "Corrections arising at a later point in time should be promptly sent to the WCA Results Team."

Reply:
![image](https://user-images.githubusercontent.com/11577241/51286230-121fcd00-19c0-11e9-89ba-335bc487da99.png)
I'm guessing this comment is not applicable anymore with a some point the highlighted text was added to the document?

Reply:
Indeed, I think the intentions of the document broadened. I agree with the suggestions made by @SAuroux.

Reply:
I agree too!

Reply:
Same here!

Reply:
Suggestion committed.